test/integration/fixtures/blocks/core__media-text.html

```diff
@@ -1,5 +1,5 @@
 <!-- wp:media-text {"mediaType":"image"} -->
-<div class="wp-block-media-text alignwide is-stacked-on-mobile">
+<div class="wp-block-media-text is-stacked-on-mobile">
```

Comment:
For the primary block fixture, I've removed the `alignwide` class as that was the default for the previous version. The current source HTML matches the current version of the block. A separate deprecation has been added to cover the deprecation of the default `wide` alignment.